.travis.yml

```diff
@@ -63,10 +62,6 @@ before_script:
   # Remember the Drupal installation path.
   - DRUPAL_DIR=$(pwd)/drupal
 
-  # Download Drupal console so we can run the test for it. Skip this for the
-  # coding standards test.
-  - test ${TEST_SUITE} == "PHP_CodeSniffer" || composer require --dev drupal/console:~1.0 --working-dir=$DRUPAL_DIR
-
```

Comment:
I think we need to do some fancy logic and if TEST_SUITE == "9.0.x" then use phpunit 7.